resource-timing/initiator-type/misc-script.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
```

Comment:
If we remove the `link` test, I think we're only left with worker tests in this file so maybe we could call this `workers.html`?